metadata.json

```diff
@@ -33,8 +33,7 @@
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "8",
-        "9"
```

Comment:
Why drop 9? It should really be the only version since 8 is EOL.

Reply:
I'm not sure why I had that in there - its gone now